modules/library/src/main/scala/zio/elasticsearch/ElasticExecutor.scala

```diff
@@ -28,6 +28,10 @@ private[elasticsearch] trait ElasticExecutor {
   def stream(request: Search): Stream[Throwable, Item]
 
   def streamAs[A: Schema](request: Search): Stream[Throwable, A]
+
+  def largeStream(request: Search): Stream[Throwable, Item]
```

Comment:
I would rename this... Maybe it makes sense for me to be `stream` and `streamUpTo10k` or something like that?

Reply:
I know we should use different naming but nothing made sense to me.

Reply:
Maybe it should be controlled trought some settings or configurations when creating executor/request. Naming it will be hard to make it have sense

Reply:
Let's activate a 🃏! @arnoldlacko

Reply:
Agreed to introduce the possibility of passing another parameter in the form of `StreamConfig.scroll.pageSize(10)...`